### ISSUE TITLE
Evaluation server, big files and manual submission

### ISSUE THREAD

**Issue by kgiannakelos** [CAPTAIN-WHU/iSAID_Devkit#25]
Hello, 

i uploaded an 88mb zip to the evaluation server but it seems that the server went down and didn't respond after a day.

The uploading speed also seems very slow. Is there any other way for uploading our results?

Maybe i can send you manually the file via email or dropbox and make the evaluation yourselves? It is important to get some immediate results.

Thanks you. 

-- UPDATE --

I uploaded a very small subset of my results and the evaluation server responded, so the format is correct.

It seems that i definitely need to do a manual submission of my results since the evaluation server times out. Could you please help me with that?

**Comment by akshitac8** [comment]
Hello @kgiannakelos, the evaluation server is maintained by @dingjiansw101 he will be able to help you solve the issues regarding the server.

**Comment by kgiannakelos** [comment]
@akshitac8 Thanks for your response. I already contacted @dingjiansw101 but the issue is not resolved. I need to do a manual submission because the server times out and cannot complete the full evaluation. 

Shall i send you the file and have you delegate the request to the evaluation server?

According to [this](https://captain-whu.github.io/iSAID/evaluation.html), it is possible to send you the results on akshita.gupta@inceptioniai.org (i assume it's you @akshitac8) right?

**Comment by kgiannakelos** [comment]
@akshitac8 Any updates on this? Can i send you my results as stated [here](https://captain-whu.github.io/iSAID/evaluation.html)?

**Comment by akshitac8** [comment]
@kgiannakelos Can you try to upload the results on the sever now?

**Comment by kgiannakelos** [comment]
@akshitac8 The upload page does not load at the moment.

**Comment by kgiannakelos** [comment]
@akshitac8 Tried again, still no response.

If i use a subset of my results (results with at least 15% confidence instead of 5%), it works but then do not get a proper mAP calculation. So i guess the size of my file is the problem. 

Can you try evaluating my file to your server? Only once would suffice.

Thank you.

**Comment by czq693497091** [comment]
> @kgiannakelos Can you try to upload the results on the sever now?

I also have the same question. Can I send you the result and you help me evaluate the result in local enviroment? Only once would suffice.
Thank you!

**Comment by akshitac8** [comment]
@czq693497091 @kgiannakelos It takes roughly about 1 hour for uploading the results on the server and during that time it doesn't show anything. So, please don't refresh the browser before that time.

**Comment by kgiannakelos** [comment]
@akshitac8 Yes i know and that's why I do not refresh the page, i just leave it as it is and still get timeout after 3-5 hours. 

Is it maybe hanging due to some other reason? Perhaps multiple concurrent requests?.

**Comment by akshitac8** [comment]
@kgiannakelos I uploaded the results from my end and it works for me. I got the results after 1.5 hours. if the issue still persists, then @dingjiansw101 is the right person to contact as he handles the server.

**Comment by kgiannakelos** [comment]
@akshitac8 What results did you upload? If you uploaded your own results then obviously they are not comparable to my results. 

I already contacted @dingjiansw101 but i didn't get any help. 

So to my understanding, a manual submission is not possible?
